WcaOnRails/app/mailers/delegate_status_change_mailer.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class DelegateStatusChangeMailer < ApplicationMailer
+  def notify_board_and_wqac_of_delegate_status_change(user_whose_delegate_status_changes, user_senior_delegate)
+    @user_whose_delegate_status_changes = user_whose_delegate_status_changes
+    @user_senior_delegate = user_senior_delegate
+    to = ["board@worldcubeassociation.org"]
+    cc = ["quality@worldcubeassociation.org"]
+    mail(
+      to: to,
+      cc: cc,
+      reply_to: user_senior_delegate.email,
+      subject: "#{user_senior_delegate.name} just changed the Delegate status of #{user_whose_delegate_status_changes.name}",
```

Comment:
This made me realize we are not consistent about how we capitalize the word "delegate" throughout the website. See https://github.com/jfly/worldcubeassociation.org/blob/efcd848621418fd54dd6d7c8be42faf2f2dd1511/WcaOnRails/config/locales/en.yml#L380 for an example.

Could you create a new issue to discuss what exactly we want to do, and then we can start working on making this consistent across our website?

Reply:
Will do!